### ISSUE TITLE
Another questions about the loss function

### ISSUE THREAD

**Issue by qinhaifangpku** [sebp/tf_autoencoder#2]
Hi~
I have noticed that you use the sigmoid cross entropy loss function to learning the autoencoder.
But if my data is a RGB image, It is impossible to use this loss. 
I wonder if my opinion is true?
